vaadin-radio-group.html

```diff
@@ -0,0 +1,170 @@
+    }
+
+    (function() {
+
+      class GroupRadioElement extends Vaadin.ThemableMixin(Polymer.Element) {
+        static get is() {
+          return 'vaadin-radio-group';
+        }
+
+        static get properties() {
+          return {
+            name: {
+              type: String,
+              reflectToAttribute: true
+            },
+            value: {
+              type: String,
+              notify: true,
+              observer: '_activeValue',
+              reflectToAttribute: true
+            },
+            required: {
+              type: Boolean,
+              reflectToAttribute: true
+            },
+            disabled: {
+              type: Boolean,
+              reflectToAttribute: true,
+              observer: '_checkDisabled'
+            }
+          };
+        }
+
+        ready() {
+          
+          super.ready();
+
+          this._addActiveListeners();
+
+          this.addEventListener('radio-changed', e => {
+            if(e.target.checked) {
+              this.querySelectorAll('vaadin-radio-button').forEach((item) => item.checked = (e.target === item) ? true : false);
+              this.value = e.target.value;
+            }
+          });
+
+        }
+
+        _checkDisabled(value) {
+
+          if(value)
+            this.removeAttribute('invalid');
+          this.querySelectorAll('vaadin-radio-button').forEach((item) => item.disabled = value);
+
+        }
+
+        _addActiveListeners() {
+
+          this.addEventListener('keydown', e => {
+            if (!this.disabled && (e.keyCode === 37 || e.keyCode === 38)) {
+              e.preventDefault();
+              if(e.target.previousElementSibling) {
+                e.target.previousElementSibling.checked = true;
+                e.target.previousElementSibling.focus();
+                this.value = e.target.previousElementSibling.value;
+              } else {
+                this.lastElementChild.checked = true;
+                this.lastElementChild.focus();
+                this.value = this.lastElementChild.value;
+              }
+            }
+            if (!this.disabled && (e.keyCode === 39 || e.keyCode === 40)) {
+              e.preventDefault();
+              if(e.target.nextElementSibling) {
+                e.target.nextElementSibling.checked = true;
+                e.target.nextElementSibling.focus();
+                this.value = e.target.nextElementSibling.value;
+              } else {
+                this.firstElementChild.checked = true;
+                this.firstElementChild.focus();
+                this.value = this.firstElementChild.value;
+              }
+            }
+          });
+
+        }
+
+        _activeValue(set) {
+
+          if(set)
+            this.removeAttribute('invalid');
+          this.querySelectorAll('vaadin-radio-button').forEach((item) => item.checked = false);
+          if(this.querySelector('vaadin-radio-button[value="' + set + '"]'))
```

Comment:
nit: consider using template literals and `${this.is}` instead of copying tag name.